fe/fe-core/src/main/java/org/apache/doris/transaction/GlobalTransactionMgr.java

```diff
@@ -198,6 +224,16 @@ public void commitTransaction(long dbId, List<Table> tableList, long transaction
         DatabaseTransactionMgr dbTransactionMgr = getDatabaseTransactionMgr(dbId);
         dbTransactionMgr.commitTransaction(tableList, transactionId, tabletCommitInfos, txnCommitAttachment);
     }
+
+    public void commitTransaction(long dbId, List<Long> transactionIds)
```

Comment:
```suggestion
    private void commitTransaction(long dbId, List<Long> transactionIds)
```

Reply:
Better to change the method name, for example, adding a prefix of suffix `2PC` to make it different from the normal `commitTransaction`. Or it is confused for code reading.

All other methods which are related to `2PC` in both GlobalTransactionMgr and DatabaseTransactionMgr need to be changed.